go/pserver/service.go

```diff
@@ -90,50 +93,58 @@ type Service struct {
 	checkpointInterval time.Duration
 	checkpointPath     string
```

Comment:
It is not related to this PR. I remember service.checkpointPath default value is `/checkpoint/`.  which we do not have permission without root privilege. Need to check it before we test.